### DISCUSSION TITLE
11.1.2023

### DISCUSSION THREAD

**Discussion by charlieforward9** [charlieforward9/ATHLEAT#47] [Team Meetings]
## Deliverables (by Nov 7)

* @PrestonGoren #14 

* @rensaketh #13 

* @charlieforward9 #19 #20 

* @ticogarcia #21
     * Decide on which Charting Library to use, share in this discussion

**Comment by ticogarcia** [comment]
Decided on Chart.js as our data visualization library. https://www.chartjs.org/docs/latest/
Looks easy to use and should be capable of plotting the graphs that we want.
